todo_app/templates/index.html

```diff
@@ -12,8 +12,14 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
-        {% for item in items %}
-          <li>{{ item.id }} - {{item.title}}</li>
+        {% for item in todo_items %}
+          <li>{{ item.id }} - {{item.title}} - {{item.status}}</li>
+          <form action="complete-item" method="post" class="frmItem">
+            <div class="form-example">
+              <input type="text" name="todo-id" id="todo-id" value="{{ item.id }}" hidden>
+              <input type="submit" value="Complete item">
+            </div>
+          </form>
```

Comment:
Could you change this form so that it only appears when the status isn't complete/done?